process-engine/bpmn-studio
# ✨Add Remove Button to Signal, Escalation, Error and Message Event

## What did you change?

This PR adds the possibility to remove signals, escalations, errors and messages from the diagram.

Fixes #591 

## How can others test the changes?

- checkout the branch
- `npm run build`
- `npm run electron-start-dev`
-  add e.g. a signal boundary event to a task and add some signals
- open the xml view and see that the xml has increased
- switch to the diagram and select the boundary event again
- remove one or more signals and switch back to the xml view
- see that the xml has decreased

Please test it for signals, escalations, errors and messages!

## PR-Checklist

Please check the boxes in this list after submitting your PR:
